src/http_handlers/metrics.hpp

```diff
@@ -143,10 +142,11 @@ class MetricsService {
   }
 };
 
-template <typename TSessionContext>
+// TODO: Should this be inside Database?
+// Raw pointer could be dangerous
```

Comment:
Agreed, ATM MetricsServer could outlive the database (deleted/unloaded).
Maybe should use the DatabaseHandler to provide metrics for databases which are currently loaded.

Need to discuss what `nlohmann::json AsJson(MetricsResponse response)` should return, and what depends upon it (I assume Lab).

Reply:
There is a Telemetry redesign planed. Should we postpone this fix till then?

Reply:
Agreed